1-Initial-Setup/1.7-Command-Line-Warning-Banners.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+
+@test "1.7.1 Ensure message of the day is configured properly (Automated)" {
+    run bash -c 'grep -Eis "(\\\v|\\\r|\\\m|\\\s|$(grep '\''^ID='\'' /etc/os-release | cut -d= -f2 | sed -e '\''s/"//g'\''))" /etc/motd'
+    [ "$status" -ne 0 ]
```

Comment:
`-ne 0` could lead to a false negative. Maybe `-eq 1` is more suitable?

Reply:
you're right, I'll fix that.